data-raw/hospital_bed_capacity.csv

```diff
@@ -0,0 +1,154 @@
+country_name,year,income_group,region,hospital_beds,pred_beds,beds_to_use
+Afghanistan,2015,Low income,South Asia,0.5,0.667767334,0.5
+Albania,2016,Upper middle income,Europe & Central Asia,NA,5.226680541,5.226680541
+Angola,2011,Lower middle income,Sub-Saharan Africa,NA,0.91921016,0.91921016
+Antigua and Barbuda,2015,High income,Latin America & Caribbean,NA,3.627857776,3.627857776
+Argentina,2008,Upper middle income,Latin America & Caribbean,NA,2.706176279,2.706176279
+Austria,2016,High income,Europe & Central Asia,NA,7.094084256,7.094084256
+Bahamas,2016,High income,Latin America & Caribbean,NA,3.334477454,3.334477454
+Bahrain,2016,High income,Middle East & North Africa,NA,3.605374254,3.605374254
+Bangladesh,2010,Lower middle income,South Asia,NA,1.792190019,1.792190019
+Barbados,2016,High income,Latin America & Caribbean,NA,7.092706307,7.092706307
+Belarus,2016,Upper middle income,Europe & Central Asia,NA,6.445866786,6.445866786
+Belgium,2016,High income,Europe & Central Asia,NA,5.227973953,5.227973953
+Belize,2016,Upper middle income,Latin America & Caribbean,NA,1.51488923,1.51488923
+Benin,2016,Low income,Sub-Saharan Africa,NA,1.257592328,1.257592328
+Bhutan,2016,Lower middle income,South Asia,NA,2.472514992,2.472514992
+Bolivia,2016,Lower middle income,Latin America & Caribbean,NA,1.224848965,1.224848965
+Botswana,2007,Upper middle income,Sub-Saharan Africa,2.4,2.545022486,2.4
+Brazil,2016,Upper middle income,Latin America & Caribbean,NA,2.42346711,2.42346711
+Brunei Darussalam,2016,High income,East Asia & Pacific,NA,2.686327997,2.686327997
+Bulgaria,2016,Upper middle income,Europe & Central Asia,NA,6.59524654,6.59524654
+Burkina Faso,2016,Low income,Sub-Saharan Africa,NA,1.296836767,1.296836767
+Burundi,2016,Low income,Sub-Saharan Africa,NA,1.405254262,1.405254262
+Cabo Verde,2016,Lower middle income,Sub-Saharan Africa,NA,2.218710128,2.218710128
+Cambodia,2007,Lower middle income,East Asia & Pacific,NA,1.708706085,1.708706085
+Cameroon,2016,Lower middle income,Sub-Saharan Africa,NA,1.354560419,1.354560419
+Central African Republic,2016,Low income,Sub-Saharan Africa,NA,0.644838173,0.644838173
+Chad,2016,Low income,Sub-Saharan Africa,NA,0.364916345,0.364916345
+Chile,2015,High income,Latin America & Caribbean,NA,3.506149199,3.506149199
+China,2016,Upper middle income,East Asia & Pacific,NA,6.186642526,6.186642526
+Colombia,2016,Upper middle income,Latin America & Caribbean,NA,1.821092731,1.821092731
+Comoros,2014,Lower middle income,Sub-Saharan Africa,NA,0.84988754,0.84988754
+"Congo, Dem. Rep.",2015,Low income,Sub-Saharan Africa,NA,0.788912917,0.788912917
```

Comment:
These have " in front of them - Democratic Republic of the Congo is fine. Better to not abbreviate a country as it's harder to gues what the abbreviation is than typing the full name

Reply:
Have changed country names to get rid of the relevant "(Republic of)" random brackets stuff etc etc!